src/screens/FeedScreen/Header.js

```diff
@@ -656,7 +656,7 @@ const styles = StyleSheet.create({
     fontWeight: '500'
   },
   textFollowing: {
-    color: COLORS.greySubtile1,
+    color: COLORS.gray400,
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The color for `textFollowing` has been updated from `COLORS.greySubtile1` to `COLORS.gray400`. Ensure this change is consistent with your design guidelines and maintains good contrast for readability.
```diff
- color: COLORS.greySubtile1,
+ color: COLORS.gray400,
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->